### PR TITLE
setup description

### PR DESCRIPTION
**Pull request by memsharded**
Changelog: omit
Docs: omit

@pyvers: Linux@py27, Linux@py36

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,11 +47,12 @@ def load_version():
...
 
 
-# def generate_long_description_file():
-#     import pypandoc
-#
-#     output = pypandoc.convert('README.md', 'rst')
-#     return output
+def generate_long_description_file():
+    this_directory = path.abspath(path.dirname(__file__))
+    with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
```
Comment (lasote) [inline setup.py]: It is not README.md but README.rst

Comment (memsharded) [reply setup.py]: Done

**Comment by lasote** [comment]
The rst format might be valid too.

**Comment by uilianries** [comment]
I can confirm it's now available on https://test.pypi.org/project/conan/

**Pull request merged**